server/tests/auth/test_oauth2_state_cache.py

```
@@ -0,0 +1,320 @@
+    clear_oauth2_states,
+    retrieve_oauth2_state,
+    store_oauth2_state,
+)
+
+
+# ---------------------------------------------------------------------------
+# Fixtures
+# ---------------------------------------------------------------------------
+
+
+@pytest.fixture()
```

Ruff (0.15.12)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)